public/chore.js

```diff
@@ -0,0 +1,32 @@
+function clearFields() {
+    const elements = document.getElementById('chore-form').elements;
+
+    for (i = 0; i < elements.length; i++) {
+        fieldType = elements[i].type.toLowerCase();
+        switch (fieldType) {
+        case 'text':
+            elements[i].value = '';
+            break;
+        case 'checkbox':
+            if (elements[i].checked) {
+                elements[i].checked = false;
+            }
+            break
+        default:
+            break;
+        }
+    }
+}
+
+
+const choreList = ["Make Bed", "Walk Dogs", "Clean Room", "Empty Dishwasher"];
+const list = document.createElement('ul');
+
+choreList.forEach(function (choreList) {
```

Comment:
in a forEach, the temporary variable used to represent each item in the list should have a different name than the array/list. 

For ex:
```
choreList.forEach(function (chore) {
    const li = document.createElement('li');
    li.textContent = chore;
    list.appendChild(li);
});
```

Now it reads better >
In the chore list, for each **chore**, create a line item.